projekt_do_pracy_na_labach/fiszki_backend/backend.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: example-deployment
+  namespace: lab
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: example
+  template:
+    metadata:
+      labels:
+        app: example
+    spec:
+      containers:
+      - name: backend-service
+        image: fiszki-backend:latest
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 8000
+
+##apiVersion: v1
```

Comment:
Jak coś mozesz trzymać to w jednym pliku bez komentarzy. Wystarczy że dodasz `---` po definicji deploymentu i wtedy mozesz od razu definiować kolejny zasób. kubectl podczas tworzenia ogarnie